source/manual/load-test.html.md

```diff
@@ -0,0 +1,116 @@
+---
+owner_slack: "#govuk-developers"
+title: Run a load test with k6
+section: Testing
+layout: manual_layout
+parent: "/manual.html"
+---
+
+These instructions are based on [this blog post](https://grafana.com/blog/2022/06/23/running-distributed-load-tests-on-kubernetes/).
+
+1. Log into AWS and set the context to the environment you wish to test:
+
+    ```sh
+    ENVIRONMENT=staging
+    ROLE=admin
+    AWS_REGION=eu-west-1
+
+    eval $(gds aws govuk-${ENVIRONMENT}-${ROLE} -e --art 8h)
+    kubectl config use-context $ENVIRONMENT
+    ```
+
+    > You will need to assume the `admin` role in order to deploy k6 to the cluster.
+
+1. Clone the `k6-operator` repo and make this:
+
+    ```sh
+    git clone https://github.com/grafana/k6-operator && cd k6-operator
```

Comment:
**question:** HTTPS vs SSH

Any reason for the example using HTTPS rather than SSH? I thought SSH was generally preferred, but from a quick search it sounds as though GitHub flip flop on what they recommend

Reply:
Just stole it from the linked blog.  I believe we normally suggest SSH elsewhere, so I'll use that instead.